src/Sulu/Bundle/SecurityBundle/Entity/AccessControlRepository.php

```diff
@@ -37,13 +37,18 @@ public function findByTypeAndIdAndRole($type, $id, $roleId)
         }
     }
 
-    public function findByTypeAndId($type, $id)
+    public function findByTypeAndId($type, $id, $system = null)
```

Comment:
Not sure if we should do either of those things... This class is probably not touched by anybody... If I could do it again, I would remove the interface and make that class final, because it is not meant to be overridden... Adding code like what you are suggesting is cumbersome, for not a big gain, and documenting it will bloat the `UPGRADE.md`, which will make people read it less carefully...

If the rest of the team disagrees I am find with implementing it the way you've suggested though.